recipes/comboboxes/combobox_with_popover/combobox_with_popover.vue

```diff
@@ -430,14 +432,12 @@ export default {
     },
 
     onFocusOut (e) {
-      const comboboxRefs = ['input', 'header', 'footer', 'listWrapper'];
       // Check if the focus change was to another target within the combobox component
-      const isComboboxStillFocused = e.relatedTarget === null || comboboxRefs.some((ref) => {
-        return this.$refs[ref]?.contains(e.relatedTarget);
-      }) || this.visuallyHiddenClose;
+      const isComboboxStillFocused = this.$refs?.combobox?.$el.contains(e.target);
+      if (isComboboxStillFocused) return;
 
       // If outside the combobox then close
-      if (!isComboboxStillFocused) { this.closeComboboxList(); }
+      this.closeComboboxList();
```

Comment:
This is certainly cleaner but unfortunately is not going to work. If you click anywhere within the popover with the mouse it will immediately close. This is the problem the old previous code was solving. If the click was within the open popover, or goes to "unfocused" (`e.relatedTarget === null`) by clicking a non interactive element, the popover will remain open.

Reply:
I'm sure I tested that a million times and it was working correctly, let me see if I forgot to push something.

Reply:
Don't know what happened, added back the right code, should be fixed.

Reply:
yep looks good now thanks!